jscomp/others/js.ml

```diff
@@ -96,7 +99,12 @@ type +'a undefined
   A value of this type can be either undefined or 'a. This type is equivalent to Js.Undefined.t.
 *)
 
-type +'a nullable
+type +'a nullable =
+  | Value of 'a
+  | Null [@as null]
```

Comment:
In the `Js` scope there are 2 meanings for `Value` and `Null`. And `Js.Null` is the last one listed.
Just pointing this out, and see if that's OK, and if the order is the desired one.
I guess Core will take care of these things and this stuff in `Js` will only be  a "storage" of type definitions not used directly?